.github/BOTMETA.yml

```diff
@@ -909,106 +911,100 @@ files:
     - core inventory
     - inventory
     - inventory parsing
-  $module_utils/a10.py:
+  $module_utils/network/a10:
     maintainers: ericchou1 mischapeters
     labels: networking
-  $module_utils/aci.py:
+  $module_utils/network/aci:
     maintainers: $team_aci
     labels:
     - aci
     - networking
-  $module_utils/aireos.py:
+  $module_utils/network/aireos:
     maintainers: jmighion
     labels: networking
+  $module_utils/network/aos:
+    maintainers: dgarros jeremyschulman
+    labels: networking
   $module_utils/azure_rm_common.py:
     maintainers: $team_azure
     labels:
       - azure
       - cloud
-  $module_utils/eos.py:
+  $module_utils/network/eos:
     maintainers: $team_networking
     labels: networking
-  $module_utils/aruba.py:
+  $module_utils/network/aruba:
     maintainers: jmighion
     labels: networking
-  $module_utils/asa.py:
+  $module_utils/network/asa:
     maintainers: ogenstad
     labels: networking
-  $module_utils/avi.py:
+  $module_utils/network/avi:
     maintainers: $team_avi
     labels:
     - avi
```

Comment:
Here as well, we can ditch label `avi` here.

Reply:
Fixed